sphinx/ext/imgmath.py

```diff
@@ -126,15 +130,22 @@ def compile_math(latex: str, builder: Builder) -> str:
     # build latex command; old versions of latex don't have the
     # --output-directory option, so we have to manually chdir to the
     # temp dir to run it.
-    command = [builder.config.imgmath_latex, '--interaction=nonstopmode']
+    command = [builder.config.imgmath_latex]
+    # assume that executable is on the PATH (i.e., don't handle /path/to/tectonic)
+    if builder.config.imgmath_latex not in ['tectonic']:
+        command.append('--interaction=nonstopmode')
     # add custom args from the config file
     command.extend(builder.config.imgmath_latex_args)
     command.append('math.tex')
 
     try:
         subprocess.run(command, capture_output=True, cwd=tempdir, check=True,
                        encoding='ascii')
-        return path.join(tempdir, 'math.dvi')
+        # FIXME: maybe it would be better to add a param: imgmath_dvi_extension?
```

Comment:
About adding a param, it would have made sense if we were to allow `pdf`.  Indeed in the past we tested `dvisvgm --pdf` as alternative.  But here it is only a matter of `dvi` versus `xdv` and no situation seems to lead to have to make a choice: the latex command forces that one of the two only works.  So, I don't think it needed to add a parameter.

Reply:
Indeed.